README.md

```diff
@@ -18,3 +18,32 @@ The documentation is divided into several sections:
 - [Business Cases](docs/business_cases.md)
 - [Architecture diagram](docs/architecture_diagram.md)
 - [ER Diagram](docs/er_diagram.md)
+
```

Comment:
The readme file should contain technical descriptions for other developers: how to start the application locally, how to use it, and what you need to install and run before starting the application.

Reply:
Discussed